tests/run-make/fmt-write-bloat/rmake.rs

```diff
@@ -12,24 +12,90 @@
 //! present.
 //!
 //! Some CI jobs try to run faster by disabling debug assertions (through setting
-//! `NO_DEBUG_ASSERTIONS=1`). If debug assertions are disabled, then we can check for the absence of
-//! additional `usize` formatting and padding related symbols.
+//! `NO_DEBUG_ASSERTIONS=1`). If std debug assertions are disabled, then we can check for the
+//! absence of additional `usize` formatting and padding related symbols.
+
+// ignore-tidy-linelength
 
 //@ ignore-cross-compile
 
-use run_make_support::artifact_names::bin_name;
+// NOTE: against the GCC codegen backend, the assertion
+// `object_contains_any_symbol_substring(&expect_panic_symbols, &panic_syms)` fails.
+//@ ignore-backends: gcc
```

Comment:
Remark: ignoring this test for GCC cg backend, I do not feel like investigating why https://github.com/rust-lang/rust/pull/147819#issuecomment-3416629906 when the GCC cg backend is used, we observe some of these panic symbols.

Reply:
Yeah, if the test ends up becoming flaky, I honestly rather disable or even remove this entire test. Because this test is exercising panic-machinery related optimizations through indirect implementation details, and checking for what is there and isn't there are IMO equally prone to "no longer testing what it's supposed to test". This is still the case even if we use DIA wrappers :(